monitoring/v1/types.go

```diff
@@ -287,7 +287,7 @@ type AlertRelabelConfigList struct {
 	// items is a list of AlertRelabelConfigs.
 	// +kubebuilder:validation:MinItems:=1
 	// +required
-	Items []*AlertRelabelConfig `json:"items"`
+	Items []AlertRelabelConfig `json:"items"`
```

Comment:
I think this should be optional, and not include the minitems, could you update while we are making this change please
```suggestion
	// +optional
	Items []AlertRelabelConfig `json:"items,omitempty"`
```

Reply:
Sure no problem. I had a quick skim around and it seems like
```
Items []AlertRelabelConfig `json:"items"`
```
without any further markers are very common. Should I still add `//+optional` and `omitempty`?

Reply:
Yes please, moving towards every field being marked as either optional or required. Items fields are optional, and therefore need to have omitempty